tests/degiro_connector/quotecast/test_quotecast_network.py

```diff
@@ -20,23 +22,12 @@
 # TESTS FEATURES
 @pytest.mark.network
 @pytest.mark.quotecast
-def test_chart(quotecast_connected):
+def test_chart(quotecast_connected, stock_request):
     time.sleep(random.uniform(0, 2))
 
-    request = Chart.Request()
```

Comment:
I moved this to a fixture since I wanted to reuse it on my new test. I'm not very happy with the name I chose. Any suggestions?